eth/downloader/downloader.go

```diff
@@ -317,6 +317,9 @@ func (d *Downloader) Synchronise(id string, head common.Hash, td *big.Int, mode
 	err := d.synchronise(id, head, td, mode)
 	switch err {
 	case nil:
+		select {
+		default:
+		}
```

Comment:
If find the golang selects and channels a bit confusing -- but what exactly does this piece of code do? Looks like a no-op to me?

Reply:
Sorry about this as well.  This was due to originally using a new channel, but switching to the downloader.done event removed the need for this.  Was preoccupied working with downloaderdone.